src/adzerk/response_models.rs

```diff
@@ -197,12 +197,12 @@ fn map_priority(priority_id: Option<u32>) -> u32 {
 fn get_cdn_image(full_image_path: &str) -> Result<String, ProxyError> {
     match full_image_path.parse::<Uri>()?.host() {
         Some(domain) if domain.ends_with(".zkcdn.net") || domain == "zkcdn.net" => {
-            let mut result = "https://img-getpocket.cdn.mozilla.net/direct?".to_owned();
-            form_urlencoded::Serializer::new(&mut result)
+            let result = "https://img-getpocket.cdn.mozilla.net/direct";
+            let url = form_urlencoded::Serializer::new(String::new())
                 .append_pair("url", full_image_path)
                 .append_pair("resize", "w618-h310")
                 .finish();
-            Ok(result)
+            Ok(format!("{}?{}", result, url))
```

Comment:
You can inline `result` here now – there really isn't any need to do this dynamically anymore.